doc/specs/vulkan/chapters/extensions.txt

```diff
@@ -223,6 +223,16 @@ wrapped by the validation layers.
 See the <<LoaderAndValidationLayers, "`Vulkan Loader Specification and
 Architecture Overview`">> document for additional information.
 
+[NOTE]
```

Comment:
I'm not sure why you are moving this note up sooner, and out of the Extension Dependencies section.

Reply:
I see you and @drakos-amd discussed this below.  I'll comment further below ....